testdata/project-v3-addon/README.md

```diff
@@ -0,0 +1,85 @@
+# Project Name
+// TODO(user): Add simple overview of use/purpose
+
+## Description
+// TODO(user): An in-depth paragraph about your project and overview of use
+
+## Getting Started
+In order to use this operator you will need to have a Kubernetes cluster to run against.
+If you need a local cluster for testing you can use [KinD](https://kind.sigs.k8s.io/)
+
+### Running the operator
+To run this operator locally (outside of a Kubernetes cluster):
+	
+1. Install the CustomResourceDefinitions (CRDs) into the cluster:
+	~~~shell
+	make install
+	~~~
+	
+2. Run the controller (this will run in the foreground):
+	~~~shell
+	make run
+	~~~
+	
+**NOTE:** You can also run this in one step by running: <code>make install run</code>
+
+To stop running the operator locally, you can terminate the process by pressing <code>ctrl+c</code> in the terminal window the process is running in
+
+To run this operator on the cluster:
+	
+1. Install the CRDs:
+	~~~shell
+	kubectl apply -f config/samples/
+	~~~
+	
+2. Build and push your image to the location specified by <code>IMG</code>:
+	~~~shell
+	make docker-build docker-push IMG=<some-registry>/<project-name>:tag
+	~~~
+	
+3. Deploy the controller to the cluster with the image specified by <code>IMG</code>:
+	~~~shell
+	make deploy IMG=<some-registry>/project-name>:tag
+	~~~ 
+
+### Uninstall CRDs
+To delete the CRDs from the cluster:
+~~~shell
+make uninstall
+~~~
+
+### Undeploy Controller
+If you deployed the controller on the cluster, you can undeploy it with:
+~~~shell
+make undeploy
+~~~
+
+**NOTE:** Run <code>make --help</code> for more information on all potential <code>make</code> targets
+
+More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
+
+## Contributing
+// TODO(user): Add detailed information on how you would like others to contribute to this project
+
+### How it works
+This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+
+It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) 
+which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster 
+
+## License
+/*
```

Comment:
We will need to remove the /* from the value that we pass to the boilerplate. 
we can use the strings.ReplaceAll(value, "/*", "") and strings.ReplaceAll(value, "*\", "") before pass it to the bollerplate

Reply:
Fixed in commit https://github.com/kubernetes-sigs/kubebuilder/pull/2610/commits/dee7b87ab71c0f0b09e1154c1e1abfaa9732c73c